src/MessagingInput/MessagingInput.tsx

```diff
@@ -83,6 +83,9 @@ const MessagingInputRenderFunction: React.ForwardRefRenderFunction<MessagingInpu
   const attachmentsArePresent = attachments?.length > 0;
   const [isInputActive, setIsInputActive] = useState(false);
 
+  // Update this if relevant styling changes and this heuristic no longer holds
+  const hasMultiLineText = textAreaRef.current?.textAreaHeight() > 40;
```

Comment:
I know this heuristic isn't perfect, but it was the best I could come up with for a quick, low-lift fix!

Reply:
Random thought here: what if you checked if the element `scrollHeight`  is larger than the `clientHeight`? Should hold up even if the input styling changes.

That being said, this looks okay to me as well!

Reply:
Ooh great idea! The one catch is that we also want to decrease the rounding when there are only two lines of text. In this case, there isn't a scrollbar present, so the `scrollHeight > clientHeight` heuristic won't catch that case